addons/mass_mailing_sms/tests/test_mailing_internals.py

```diff
@@ -59,3 +59,32 @@ def test_mailing_computed_fields(self):
             'contact_list_ids': [(4, self.mailing_list_1.id), (4, self.mailing_list_2.id)]
         })
         self.assertEqual(literal_eval(mailing.mailing_domain), [('list_ids', 'in', (self.mailing_list_1 | self.mailing_list_2).ids)])
+
+    @users('user_marketing')
+    def test_mailing_get_replacements_placeholders(self):
```

Comment:
:socks: just don't forget to update this one if we update the method naming (that's quality riviou, thanks tde)